docs/deployment/docker-setup.md

```diff
@@ -1,9 +1,13 @@
 # Docker
-## What is docker
-TODO
+## Docker in really short
+In really short, docker is like a virtual machine but without the kernel emulation. Every container share the host kernel. For rtc, we use linux containers that require a linux kernel.
+
+On Windows or Mac, linux containers run inside a virtual machine. This Virtual machine provides the linux kernel. On linux, it's the host kernel that is shared with the containers.
 
 ## Why docker
-TODO
+One of the biggest benefits is time. A container starts in less than one second. This let's us use a multiple of them for tiny tasks. It also provides a really short restart for services.
```

Comment:
+One of the biggest benefits is boot time. A container starts in less than one second. This lets us to use a multiple of them for tiny tasks. It also provides a really short restart time for services.

Reply:
In next commit